atm.js

```diff
@@ -1,21 +1,71 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
```

Comment:
I believe you were referencing not being able to test values in the console in your PR description. Important to remember that by placing things in `$(document).ready(function(){...})` we are removing them from the global scope.